scripts/json/updateNode.json

```diff
@@ -0,0 +1,11 @@
+{
+  "host_info": {
+    "node_id" : "9192a4d4-ffff-4ece-b3f0-8d36e3d85002",
+    "local_ip": "10.213.43.161",
+    "mac_address": "90:17:ac:c1:34:64",
+    "node_name": "node2",
+    "server_port": 8080,
+    "veth": "eth0",
+    "host_dvr_mac": "74-70-FD-73-09-07"
+  }
+}
```

Comment:
This looks like Kiran's change checked in a while ago :-)

Reply:
This is my change. I added ncmtest.sh in scripts which uses this file.